ui/app/AppLayouts/Wallet/HistoryTab.qml

```diff
@@ -28,22 +43,19 @@ Item {
             }
         }
 
-
         TransactionModal{
           id: transactionModal
         }
 
         Item {
-
-          Rectangle {
-            id: assetIcon
-            color: "gray"
-            width: 40
-            height: 40
-            anchors.left: parent.left
-            anchors.top: parent.top
-            anchors.topMargin: 12
-            radius: 50
+          Image {
+              id: assetIcon
+              width: 40
+              height: 40
+              source: "../../img/tokens/" + (transactionListItem.symbol != "" ? transactionListItem.symbol : "ETH") + ".png"
+              anchors.left: parent.left
+              anchors.top: parent.top
+              anchors.topMargin: 12
```

Comment:
we need a fallback here as well, for the tokens we know their symbols but don't have images

```qml
                onStatusChanged: {
                    if (assetInfoImage.status == Image.Error) {
                        assetInfoImage.source = "../../img/tokens/0-native.png"
                    }
                }
```


https://github.com/status-im/nim-status-client/blob/master/ui/app/AppLayouts/Wallet/AssetsTab.qml#L25